### PR TITLE
feat: Credit Class Page "Additional Info" improvements

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1756

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-1980--regen-registry.netlify.app/credit-classes/C62

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts [hunk 1]
```diff
@@ -24,4 +24,22 @@ export interface CreditClassMetadataLD {
   'regen:carbonOffsetStandard'?: CompactedNameUrl;
   'regen:tokenizationSource'?: string;
   'regen:certifications'?: Certification[];
+  'regen:coBenefits'?: string[];
+  'regen:measuredGHGs': string[];
+  'regen:bufferPoolAccounts': {
+    // We probably want to simplify this to be just a @list instead,
+    // but keeping as is, until C04 class metadata is updated accordingly.
```
Comment (ryanchristo) [inline web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts:31]: Should we create a followup task blocked by https://github.com/regen-network/regen-registry-standards/pull/57?

File: web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts [hunk 2]
```diff
@@ -24,4 +24,22 @@ export interface CreditClassMetadataLD {
...
+    '@type': 'schema:ItemList';
+    'schema:itemListElement': BufferPoolAccount[];
+  };
 }
+
+export type BufferPoolAccount = {
+  'schema:name': string;
+  // Keeping both regen:walletAddress and regen:address for now,
+  // until C04 class metadata gets fixed.
+  // But ultimately, we should just use regen:address.
```
Comment (ryanchristo) [inline web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts:41]: Also relevant to https://github.com/regen-network/regen-registry-standards/pull/57

**Review by ryanchristo — APPROVED** [review]
tACK

- [x] "MEASURED GHGS"
- [x] "CO-BENEFITS"
- [x] AGGREGATED PROJECTS is "True"
- [x] schema:duration is rendered as text (e.g. 10 years)
- [x] list height greater than paragraph line height
- [x] custom buffer pool account display with name and link

File: web-registry/src/components/molecules/MetaDetail/MetaDetail.BaseValue.tsx [hunk 3]
```diff
@@ -7,34 +9,49 @@ import { formatDate, formatNumber } from 'web-components/lib/utils/format';
...
   let formattedValue: string | undefined;
   const isNumber = typeof value === 'number';
   const isString = typeof value === 'string';
+  const isBoolean = typeof value === 'boolean';
   if (isString || isNumber) {
     if (isNumber) {
       formattedValue = formatNumber({ num: value });
     } else if (rdfType === 'xsd:date') {
       formattedValue = formatDate(value);
+    } else if (rdfType?.includes('Duration')) {
```
Comment (blushi) [inline web-registry/src/components/molecules/MetaDetail/MetaDetail.BaseValue.tsx:36]: currently just "Duration" inclusion checking this but ultimately once this: https://github.com/regen-network/regen-registry-standards/pull/57#discussion_r1273270388 is fixed, we should check for `schema:Duration` equality

File: web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts (hunk 1 above)
Comment (blushi) [reply web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts:31]: done: https://github.com/regen-network/regen-web/issues/1983

**Pull request merged**